app/models/collection.rb

```diff
@@ -321,28 +315,44 @@ def gift_notification
     self.collection_profile.gift_notification || (parent ? parent.collection_profile.gift_notification : "")
   end
 
-  def moderated? ; self.collection_preference.moderated ; end
-  def closed? ; self.collection_preference.closed ; end
-  def unrevealed? ; self.collection_preference.unrevealed ; end
-  def anonymous? ; self.collection_preference.anonymous ; end
-  def challenge? ; !self.challenge.nil? ; end
+  def moderated?() = self.collection_preference.moderated
+
+  def closed?() = self.collection_preference.closed
+
+  def unrevealed?() = self.collection_preference.unrevealed
+
+  def anonymous?() = self.collection_preference.anonymous
+
+  def challenge?() = !self.challenge.nil?
 
   def gift_exchange?
-    return self.challenge_type == "GiftExchange"
+    self.challenge_type == "GiftExchange"
   end
+
   def prompt_meme?
-    return self.challenge_type == "PromptMeme"
+    self.challenge_type == "PromptMeme"
   end
 
-  def get_maintainers_email
-    return self.email if !self.email.blank?
-    return parent.email if parent && !parent.email.blank?
```

Comment:
More annoying things, sorry.

These two lines of code are also needed for all the changed collection email sending. So if the collection email or parent collection email are set, send an untranslated email there. If not, send individual emails to all maintainers translated to the language they have selected.